packages/fxa-settings/src/pages/AccountRecoveryConfirmKey/en.ftl

```diff
@@ -0,0 +1,22 @@
+## AccountRecoveryConfirmKey page
```

Comment:
Oh good to know, I didn't know this!

Reply:
I don't think any of our en.ftl files have an empty line at the top since it gets added in by default by the grunt concat task. Might not be necessary unless we're adding a group comment right after another line?

Reply:
@vpomerleau @millsoper 
Looks like that is in fact the case. Please ignore this comment then!

Reply:
Still good to know if we ever add a group comment within the file. Thanks @bcolsson!